### PR TITLE
changes for color error and non animated icon proposal

### PR DESCRIPTION
**Pull request by EduardoHenao**

### CODE REVIEW

**Comment by EduardoHenao** [comment]
changes in naming are breaking, but correct. I leave the naming changes to you.

File: lib/floating_action_bubble.dart [hunk 1]
```diff
@@ -3,11 +3,11 @@ import 'package:flutter/material.dart';
 class Bubble {
   const Bubble(
       {@required this.title,
-      @required this.titleStyle,
-      @required this.iconColor,
-      @required this.bubbleColor,
-      @required this.icon,
-      @required this.onPress});
+        @required this.titleStyle,
+        @required this.iconColor,
```
Comment (EduardoHenao) [inline lib/floating_action_bubble.dart:7]: don't mind my formatting

Comment (Darshan0) [reply lib/floating_action_bubble.dart:7]: This is good formatting sir . will merge this. thank you

**Comment by stact** [comment]
The sample needs to be changed too
icon not exists anymore replaced by animatedIconData

**Pull request merged**